What is the purpose of `storage.removeEmeSessions()` ? When should it be called ?

**Have you read the [Tutorials](http://shaka-player-demo.appspot.com/docs/api/tutorial-welcome.html)?**
Yes, the tutorials do not make use of `storage.removeEmeSessions()`

**Have you read the [FAQ](https://bit.ly/ShakaFAQ) and checked for duplicate open issues?**
Yes, no mention of `storage.removeEmeSessions()` there also.

**If the question is related to FairPlay, have you read the tutorial?**
N/A

**What version of Shaka Player are you using?**
Latest version v4.14.8

**What browser and OS are you using?**
Electron v35.1.4 with support for Google Widevine CDM (VMP enabled), that contains Chrome 134.0.6998.179
OS is Windows 11, but the question is not related to the OS.

**Please ask your question**
I have some users facing these errors in our Electron-based app, that is using offline storage of shaka player :

```
./logs/20250320T142322/renderer.log:[RENDERER] [2025-03-20 15:23:48:2348]
[ERROR] [VideoPlayerImpl] A shaka player error occurred during playback:
Severity=CRITICAL(2) Category=DRM(6) Code=FAILED_TO_CREATE_SESSION(6005)
Data=["Failed to execute 'load' on 'MediaKeySession': Rejected with system code (70)"]
```

From Widevine developer documentation (https://developers.google.com/widevine/drm/feature/browser-error-codes?hl=en#70_-_ksessionalreadyinuse), system code 70 means "kSessionAlreadyInUse", which is quite self-explanatory...

I have not yet determined the cause of this error, but my question is related to the `storage.removeEmeSessions()` API. This is a public API documented like this :  `Removes any EME sessions that were not successfully removed before. This returns whether all the sessions were successfully removed.`

It is not used in the offline storage tutorial.

So my questions are :

- What is the purpose of this API ?
- Why was it introduced ?
- What are the cases/situations it tries to solve ?
- In which cases/situations should I call it ?
- Should I always call it ?

Thank you for your help !

PS: I can submit a PR if you feel the documentation or tutorials can be improved w.r.t. this API.

--- a/lib/offline/storage.js
+++ b/lib/offline/storage.js
@@ -1120,8 +1120,23 @@ shaka.offline.Storage = class {
   }
 
   /**
-   * Removes any EME sessions that were not successfully removed before.  This
-   * returns whether all the sessions were successfully removed.
+   * Removes any orphaned EME sessions.
+   *
+   * When DRM content is stored offline with a persistent session, the DRM
+   * session ID is stored in the offline content database. When DRM content
+   * gets deleted from the database, EME is asked to release the underlying
+   * session attached to that ID.
+   *
+   * If for some reason that fails, or if the browser is closed before that
+   * asynchronous process is completed, session IDs must still be tracked in
+   * the database. Otherwise, they would get orphaned and there would be no way
+   * to discover them or clean them up.
+   *
+   * This method will clean up any orphaned, persistent DRM sessions that the
+   * database is still tracking. It should be called on application startup,
+   * and will do nothing if there are no orphaned DRM sessions. It returns a
+   * Promise that resolves to true if all the sessions were successfully
+   * removed, or false if there are still sessions remaining.
    *
    * @return {!Promise<boolean>}
    * @export

